pxr/base/tf/testenv/weakPtr.cpp

```diff
@@ -124,6 +124,24 @@ static void _TestComparisons()
     TF_AXIOM( !(NULL == x) );
 }
 
+static void _TestHash(){
+    TF_AXIOM(TfHash()(MonkeyInterfaceWeakPtr()) == TfHash()(MonkeyInterfaceWeakPtr(nullptr)));
```

Comment:
WBN to follow formatting in this file and put brace on new line. Also, I think the first line probably violates our 80 char width guideline?